.gitignore

```diff
@@ -28,11 +28,6 @@ npm-debug.log
 # The directory NPM downloads your dependencies sources to.
 /assets/node_modules/
 
-# Since we are building assets from assets/,
-# we ignore priv/static. You may want to comment
-# this depending on your deployment strategy.
-/priv/static/
```

Comment:
I feel like this is going to come back to bite us. really don't want to be committing this stuff.

How do you feel about abusing webpack's build and generating our swagger file into `assets/public/swagger.json`

From there, our normal existing build process will take over and webpack will pull it into `assets/build` which then gets pulled into our `priv/static` on release

Reply:
it even looks like we might be able to specify where this json file comes from in other parts of the configuration? can it just be in `lib/swagger.json`?

Reply:
I just tried moving it to `assets/public` and it's not as simple as it seems due to a combination of release process dirs (need to use `:code.lib_dir(:pay, :assets)` but it doesn't work as you'd expect), the dev process live reload dirs and tests so just keeping where it is for now. I reverting the `.gitignore` change and instead put a `.gitignore` file inside `priv/static` so that everything gets ignored except the `swagger.json` file. Happy to revisit it later, but for now it works and is a bit tidier at least.